config/environment.js

```diff
@@ -16,6 +16,7 @@ module.exports = function (environment) {
         Date: false,
       },
     },
+    environmentName: "{{ENVIRONMENT_NAME}}",
```

Comment:
Maybe you can also add a section to the readme about how to configure this? [Similar to how GN does it](https://github.com/lblod/frontend-gelinkt-notuleren#configuration-via-docker-environment-variables)?

Reply:
Mmm the thing is that I checked and this repo doesn't have any readme, do you know if there's some documentation elsewhere or should I start the readme just with that?

Reply:
I don't think there are other docs, no. Maybe just add the environment variable part for now since that might affect how the app is deployed? But maybe documentation isn't needed at all so I'll leave it up to @nvdk :smile:.

Reply:
good suggestion on adding a readme :). I think we can base ourselves quite a bit on the one from GN?